x-pack/plugins/observability/public/components/burn_rate_rule_editor/budget_consumed.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFieldNumber, EuiFormRow, EuiIconTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { ChangeEvent, useState } from 'react';
+import numeral from '@elastic/numeral';
+
+interface Props {
+  initialBurnRate?: number;
+  errors?: string[];
+  onChange: (burnRate: number) => void;
+  longWindowInHours: number;
+  timeWindowInHours: number;
+}
+
+export function BudgetConsumed({
+  onChange,
+  initialBurnRate = 1,
+  longWindowInHours,
+  timeWindowInHours,
+  errors,
+}: Props) {
+  const [budgetConsumed, setBudgetConsumed] = useState<number>(
+    ((initialBurnRate * longWindowInHours) / timeWindowInHours) * 100
+  );
+  const hasError = errors !== undefined && errors.length > 0;
+
+  const onBurnRateChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const value = Number(event.target.value);
+    setBudgetConsumed(value);
+    const burnRate = timeWindowInHours * (value / 100 / longWindowInHours);
+    onChange(burnRate);
```

Comment:
Maybe rename this to onBurnRateChange to make it explicit what is this callback for